cooperator_portal/controllers/portal.py

```diff
@@ -75,23 +75,21 @@ def _prepare_portal_layout_values(self):
         return values
 
     def details_form_validate(self, data):
-        error, error_message = super(
-            CooperatorPortalAccount, self
-        ).details_form_validate(data)
-        sub_req_obj = request.env["subscription.request"]
+        error, error_message = super().details_form_validate(data)
+        sub_req_model = request.env["subscription.request"]
         iban = data.get("iban")
-        valid = sub_req_obj.check_iban(iban)
+        valid = sub_req_model.check_iban(iban)
 
         if not valid:
             error["iban"] = "error"
-            error_message.append(_("You iban account number is not valid"))
+            error_message.append(_("The IBAN account number is not valid."))
```

Comment:
Prefer "Provided IBAN is not valid." for consistency with cooperator_website's controller ?

Reply:
hum… good point! why is it defined in 2 different places? i’m going to check that.